### PR TITLE
Add support for running many projects at the same time

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: infrastructure/docker/docker-compose.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ services:
             - "8080:8080"
```
Comment (jderusse) [inline infrastructure/docker/docker-compose.yml]: > Note: Port mapping is incompatible with network_mode: host

from https://docs.docker.com/compose/compose-file/

Comment (lyrixx) [reply infrastructure/docker/docker-compose.yml]: thx :+1:

File: infrastructure/docker/docker-compose.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ services:
...
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
+        network_mode: "host"
```
Comment (jderusse) [inline infrastructure/docker/docker-compose.yml]: I'm not sure exposing the router to local machine is a good thing

**Comment by jderusse** [comment]
This is how I use everal project within the same traefik:
```yaml
# /router/docker-compose.yml
networks:                       
  http:                     
    external:                                           
      name: http
services:
  proxy:
    networks:
      - http
```
```yaml
# /project/a/docker-compose.yml
networks:                       
  http:                     
    external:                                           
      name: http

services:
  app:
    image: nginx
    networks:
    - proxy
    - default
```

File: infrastructure/docker/docker-compose.yml (hunk 2 above)
Comment (lyrixx) [reply infrastructure/docker/docker-compose.yml]: Why? I was already the case before that :)

**Comment by lyrixx** [comment]
@jderusse But this means having a project with only the router? And it looke like there is a typo in your code. you have `name: http` VS `networks: [proxy, default]`

File: infrastructure/docker/docker-compose.yml (hunk 2 above)
Comment (jderusse) [reply infrastructure/docker/docker-compose.yml]: indeed... I forgot about port mapping :)

File: infrastructure/docker/docker-compose.yml [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
 services:
     router:
         build: services/router
-        ports:
-            - "80:80"
-            - "443:443"
-            - "8080:8080"
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
+        network_mode: "host"
```
Comment (Shine-neko) [inline infrastructure/docker/docker-compose.yml]: :/

Comment (lyrixx) [reply infrastructure/docker/docker-compose.yml]: Can you elaborate ?

**Pull request merged**